searxng/searxng
# Odysee: time range and languages support

## What does this PR do?
* Odysee: support for time ranges
* Odysee: support for searching by language

## Why is this change important?
* This makes the Odysee engine feature-complete

## How to test this PR locally?
* !odysee :es test
* !odysee :de test
-> Compare the results

!odysee and set a time range filter -> Observe filtered results

## Related issues
closes #2706
